test/quicer_SUITE.erl

```diff
@@ -2279,6 +2284,175 @@ tc_event_start_compl(Config) ->
   quicer:close_connection(Conn),
   ok.
 
+tc_direct_send_over_conn(Config) ->
+  Port = select_port(),
+  application:ensure_all_started(quicer),
+  ListenerOpts = [{allow_insecure, true}, {conn_acceptors, 32}
+                 | default_listen_opts(Config)],
+  ConnectionOpts = [ {conn_callback, quicer_server_conn_callback}
+                   , {stream_acceptors, 32}
+                   | default_conn_opts()],
+  StreamOpts = [ {stream_callback, quicer_echo_server_stream_callback}
+               | default_stream_opts() ],
+  Options = {ListenerOpts, ConnectionOpts, StreamOpts},
+  ct:pal("Listener Options: ~p", [Options]),
+  {ok, _QuicApp} = quicer:start_listener(mqtt, Port, Options),
+  {ok, Conn} = quicer:async_connect("localhost", Port,
+                                    [{param_conn_disable_1rtt_encryption, true} |
+                                     default_conn_opts()]),
+  %% Stream 1 enabled
+  {ok, Stm} = quicer:start_stream(Conn, [{active, true}, {quic_event_mask, ?QUICER_STREAM_EVENT_MASK_START_COMPLETE}]),
+  %% Stream 1 disabled
+  {ok, 5} = quicer:async_send(Stm, <<"ping1">>),
```

Comment:
For my own information: why does sending this message disables stream 1?

Reply:
copied from somewhere, I will remove it.